.envrc

```diff
@@ -117,7 +119,7 @@ export NODE_OPTIONS=--max-old-space-size=4096
 export SENTRY_UI_HOT_RELOAD=1
 
 ## Notify of reporting to Sentry
-if [ "$SENTRY_DEVENV_NO_REPORT" ]; then
+if [ -n "${SENTRY_DEVENV_NO_REPORT+x}" ]; then
```

Comment:
This will now work with `SENTRY_DEVENV_NO_REPORT=something` as well as `SENTRY_DEVENV_NO_REPORT=`

Reply:
Will this report or not report if we have `SENTRY_DEVENV_NO_REPORT=`?

Reply:
Bash's syntax's so unnecessary unnatural hard to read!

```
❯ SENTRY_DEVENV_NO_REPORT= ./foo.sh
+ [[ -n x ]]
+ echo 'No development environment errors will be reported (since you'\''ve defined SENTRY_DEVENV_NO_REPORT).'
No development environment errors will be reported (since you've defined SENTRY_DEVENV_NO_REPORT).

sentry on  armenzg/delete-file [$?] via ⬢ v12.19.0 via 🐍 v3.6.13 (.venv)
❯ SENTRY_DEVENV_NO_REPORT=1 ./foo.sh
+ [[ -n x ]]
+ echo 'No development environment errors will be reported (since you'\''ve defined SENTRY_DEVENV_NO_REPORT).'
No development environment errors will be reported (since you've defined SENTRY_DEVENV_NO_REPORT).

sentry on  armenzg/delete-file [$?] via ⬢ v12.19.0 via 🐍 v3.6.13 (.venv)
❯ ./foo.sh
+ [[ -n '' ]]
+ echo 'Development errors will be reported to Sentry.io. If you wish to opt-out, set SENTRY_DEVENV_NO_REPORT as an env variable.'
Development errors will be reported to Sentry.io. If you wish to opt-out, set SENTRY_DEVENV_NO_REPORT as an env variable.
+ export SENTRY_DSN=https://23670f54c6254bfd9b7de106637808e9@o1.ingest.sentry.io/1492057
+ SENTRY_DSN=https://23670f54c6254bfd9b7de106637808e9@o1.ingest.sentry.io/1492057
```